app/controllers/render-tree.js

```diff
@@ -2,16 +2,44 @@ import Ember from "ember";
 import escapeRegExp from "ember-inspector/utils/escape-reg-exp";
 import debounceComputed from "ember-inspector/computed/debounce";
 
-const { computed, isEmpty, Controller, inject: { controller } } = Ember;
+const { computed, isEmpty, Controller, inject: { controller, service } } = Ember;
 const { and, equal, filter } = computed;
 const get = Ember.get;
+let LOCAL_STORAGE_SUPPORTED;
+try {
+  LOCAL_STORAGE_SUPPORTED = !!window.localStorage;
+} catch (e) {
+  // Security setting in chrome that disables storage for third party
+  // throws an error when `localStorage` is accessed.
+  LOCAL_STORAGE_SUPPORTED = false;
+}
```

Comment:
Can you make this make this a class property on the local storage service since we're using it in two places now?

```js
// app/services/local-storage.js
LocalStorage.reopenClass({
  SUPPORTED: LOCAL_STORAGE_SUPPORTED
});
```

```js
// app/controllers/render-tree.js, app/components/x-list.js
import LocalStorageService from 'ember-inspector/services/local-storage';
storage: service(`storage/${LocalStorageService.SUPPORTED ? 'local' : 'memory'}`)
```